src/core/psi_container.jl

```diff
@@ -463,6 +463,16 @@ function get_parameter_array(psi_container::PSIContainer, ref)
     return get_parameter_array(get_parameter_container(psi_container, ref))
 end
 
+function get_model_duals(op::PSIContainer, cons::Vector{Symbol})
```

Comment:
Also, duplicate of https://github.com/NREL/PowerSimulations.jl/blob/e2cd5f2df16799ed89496b4194cc9cead0716991/src/core/psi_container.jl#L550